src/melee/ft/ft_unknown_006.c

```diff
@@ -0,0 +1,718 @@
+#include <ft_unknown_006.h>
+
+#include "fighter.h"
+#include "ftcommon.h"
+
+extern HSD_GObjList* lbl_804D782C;
+extern s32 func_80023220(s32);
+extern s32 func_800230C8(s32, s32*, s32*);
+extern s32 func_80023130();
+extern s32 func_800233EC(s32);
+extern s32 func_800C06B4(struct _Fighter*);
+
+s32 func_800877F8(HSD_GObj* fighterObj, s32 arg1)
+{
+    Fighter* fp = getFighter(fighterObj);
+    if (fp->input.x65C_heldInputs & arg1) {
+        return 1;
+    } else {
+        return 0;
```

Comment:
Would prefer to see ternary operators for these with `TRUE` and `FALSE`, or a macro to take care of it.